.config/s3.config.php

```diff
@@ -0,0 +1,20 @@
+<?php
+if (getenv('OBJECTSTORE_S3_HOST')) {
```

Comment:
S3_HOST isn't one of the required parameters. Might be better to key off of bucket name here to know we're in this.

Reply:
Requiring `S3_HOST` makes sense to me. How would Nextcloud know how to access the bucket without knowing where it's hosted?

Reply:
It's automatic/computed for the standard S3 endpoints based on account key credentials and configured/default region of the SDK configuration. 

Ultimately, in the nextcloud docs, its not a required parameter, but this will promote it to required for the container to work. In 19 and earlier, bucket, key & secret are required. In 20 and later, only bucket name is required.

Reply:
https://github.com/nextcloud/server/blob/master/lib/private/Files/ObjectStore/S3ConnectionTrait.php#L63-L80

Reply:
Ok, makes sense